ee/billing/quota_limiting.py

```diff
@@ -56,6 +57,7 @@ class QuotaLimitingCaches(Enum):
     QuotaResource.EVENTS: 0,
     QuotaResource.RECORDINGS: 1000,
     QuotaResource.ROWS_SYNCED: 0,
+    QuotaResource.FEATURE_FLAGS: 0,  # TODO: should we have a buffer here?
```

Comment:
I'm open to calling this `feature_flags_evaluated` instead, that might be more clear (I like how `rows_synced` is more obvious than just `events`.